homeassistant/components/tuya/sensor.py

```diff
@@ -729,6 +729,22 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             state_class=SensorStateClass.MEASUREMENT,
         ),
     ),
+    # Curtain
+    # https://developer.tuya.com/en/docs/iot/s?id=K9gf48qy7wkre
+    "cl": (
+        TuyaSensorEntityDescription(
+            key=DPCode.COUNTDOWN_LEFT,
+            name="Remaining Time of Countdown",
+            entity_category=EntityCategory.CONFIG,
+            icon="mdi:timer-outline",
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TIME_TOTAL,
+            name="Total Time of Opening and Closing Once",
```

Comment:
That is a bit long 😓

Reply:
It will change to `Last Operation Duration`.